Calibration/HcalCalibAlgos/plugins/HcalHBHEMuonAnalyzer.cc

```diff
@@ -314,7 +316,7 @@ void HcalHBHEMuonAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSet
   const CaloTopology *caloTopology = theCaloTopology.product();
   
   edm::ESHandle<HcalTopology> htopo;
-  iSetup.get<HcalRecNumberingRecord>().get(htopo);
+  iSetup.get<IdealGeometryRecord>().get(htopo);
```

Comment:
@bsunanda IIUC this is the outcome of the HCAL topology re-assessment you performed in 76X with HCAL parameters.